src/types.ts

```diff
@@ -4324,6 +4324,14 @@ export interface StayComp extends Comp {
 	 * If the obj should not be destroyed on scene switch.
 	 */
 	stay: boolean,
+	/**
+	 * Scenes that either destroy the object or stay (depending on sceneStay value)
+	 */
+	scenes: string[],
+	/**
+	 * Whether or not the object stays on a scene or gets destroyed on a scene
```

Comment:
Maybe `sceneStay` isn't needed. Like if you add the scenes props, it's because you want only stay in that scenes, and if not, simple stay in all scenes :)